inst/migrate/1.1.25.R

```diff
@@ -0,0 +1,12 @@
+migrate <- function(data, path, config) {
```

Comment:
Do you need some test of this migration? It looks like there is in `test-migrate.R` or most of the previous migrations.

Reply:
Good point, I'll add one